base/sysinfo.jl

```diff
@@ -200,7 +212,19 @@ function _cpu_summary(io::IO, cpu::AbstractVector{CPUinfo}, i, j)
     end
     println(io)
 end
+"""
+    Sys.cpu_summary()
```

Comment:
```suggestion
    Sys.cpu_summary(io::IO=stdout, cpu::AbstractVector{CPUinfo}=cpu_info())
```
This way you can document the argument types and defaults directly on the first line.

See also the [documentation style guide](https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation).